strawberry_django/type.py

```diff
@@ -119,19 +120,19 @@ def get_fields(django_type):
 @dataclasses.dataclass
 class StrawberryDjangoType:
     origin: Any
-    model: Any
+    model: Type[django.db.models.Model]
```

Comment:
In `strawberry-django-plus`, which I'm planning on merging here, I made this a generic type to be able to specialize this: https://github.com/blb-ventures/strawberry-django-plus/blob/main/strawberry_django_plus/type.py#L338

What do you think about doing the same here?

Reply:
Help me understand, for this use case, what's the difference between  `Type[django.db.models.Model]` and `Type[_M]`? They're both tied to the `Model` class, right?

Reply:
They are both bound to model, the case there is that I transformed `StrawberryDjangoType` into a generic class which can be specialized. e.g. `foo = StrawberryDjangoType[FooBarModel]` would make accessing `foo.model` to be `FooBarModel` instead of simply `Model`

Reply:
https://docs.python.org/3/library/typing.html#typing.Generic

Reply:
Okay, I'll make that change.

Reply:
See https://github.com/strawberry-graphql/strawberry-graphql-django/pull/162/commits/855d92d48af4f58a695657eda1c1c1775cd08e86

Reply:
Thank you! :)